spec/acceptance/odoo_spec.rb

```diff
@@ -6,9 +6,9 @@ def odoo_supported_versions
   {
     'Debian' => {
       '9'  => ['11.0', '12.0'],
-      '10' => ['11.0', '12.0', '13.0', '14.0', '15.0', '16.0'],
-      '11' => ['14.0', '15.0', '16.0', 'system'],
-      '12' => ['14.0', '15.0', '16.0', '17.0'],
+      '10' => ['11.0', '12.0', '13.0', '14.0', '15.0', '16.0', '17.0'],
+      '11' => ['14.0', '15.0', '16.0', '17.0', 'system'],
```

Comment:
Reverting the `17.0` here to match `manifest/init.pp` should make CI happy again :tada: